pkg/telemetry/prometheus/node.go

```diff
@@ -95,6 +95,10 @@ func GetUpdatedNodeStats(prev *livekit.NodeStats, prevAverage *livekit.NodeStats
 		return nil, false, err
 	}
 
+	memoryLoad, _ := getMemoryStats()
+	// On MacOS, get "\"vm_stat\": executable file not found in $PATH" although it is in /usr/bin
+	// So, do not error out. Use the information if it is available.
+
```

Comment:
yeah, strange bit here. In my local dev, this is erroring out although `vm_stat` is in the path. So, just ignoring any errors in this path.